plugins/woocommerce-blocks/assets/js/blocks/product-gallery/frontend.tsx

```diff
@@ -82,6 +85,22 @@ const productGallery = {
 			const context = getContext();
 			selectImage( context, 'previous' );
 		},
+		onThumbnailKeyDown: ( event: KeyboardEvent ) => {
+			const context = getContext();
+			if ( event.code === 'Enter' ) {
```

Comment:
I think we should also listen for `NumpadEnter` and `Space` keys, both of them can also be used to trigger a button. Same in line 95 and line 100.

And maybe adding `event.preventDefault();` to prevent <kbd>Space</kbd> from scrolling down.

Reply:
Great suggestions! I just included these keys in https://github.com/woocommerce/woocommerce/pull/44236/commits/40bf0831ae875604a6b2dd2b8c8f61e858ff120c